recipes/gi-docgen/meta.yaml

```diff
@@ -1,50 +0,0 @@
-{% set name = "gi-docgen" %}
```

Comment:
Please rebase or, if that is easier, resubmit your recipe without deleting what was already there. The infrastructure handles this automatically and deleting them manually may cause problems.

Reply:
I only did this because of the linter statement at https://github.com/conda-forge/staged-recipes/pull/18124#issuecomment-1098783435. I will continue as proposed.